### PR TITLE
feat: Added Toasts and Profile page

### PR DESCRIPTION
**Pull request by jalajcodes**
Files to review:
```
frontend/src/Contexts/toastContext.js
frontend/src/Pages/Profile.js
```

Live: https://max-store-app.herokuapp.com

### CODE REVIEW

File: frontend/src/Components/Styles/toast.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+@keyframes slide-in-right {
+  0% {
+    opacity: 0;
+    transform: scale(0.8) translateX(100%);
+  }
+  100% {
+    opacity: 1;
+    transform: scale(1) translateX(0);
```
Comment (MayankKumar10) [inline frontend/src/Components/Styles/toast.scss:30]: make it a helper function so you don't have to repeat it.

**Review by MayankKumar10 — APPROVED** [review]
Just a suggestion look into these, everything else looks good to me you can merge it.

File: frontend/src/Pages/Profile.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  if (!userState.userInfo.token) {
+    return <div>You are not logged in</div>;
+  }
+
+  return (
+    <div className="profile-page">
+      <div className="user-info">
+        <div className="card horizontal text-only">
+          <span className="badge badge-warning">
+            {userState.userInfo.isAdmin ? "Admin User" : "Basic User"}
```
Comment (anik31) [inline frontend/src/Pages/Profile.js:30]: appreciation - liked that you're adding an admin user & basic user

File: frontend/src/Contexts/toastContext.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      {children}
+
+      {toasts.length > 0 && (
+        <ul className={`stacked-list toast-list`}>
+          {toasts.map(({ id, message, type }) => (
+            <li key={id} className="toast-item">
+              <Toast
+                message={message}
+                type={type}
+                removeToast={() => removeToast(id)}
```
Comment (anik31) [inline frontend/src/Contexts/toastContext.js:38]: appreciation - nice implementation of toast, you have added stacked list as well

**Review by anik31 — APPROVED** [review]
Good work, code is clean & well structured. Toast functionality is working perfectly in the preview.

**Pull request merged**